iocore/net/BIO_fastopen.cc

```diff
@@ -27,13 +27,26 @@
 
 #include "BIO_fastopen.h"
 
+#if OPENSSL_VERSION_NUMBER < 0x10100000L
+#define BIO_set_data(a, ptr) ((a)->ptr = (ptr))
+#define BIO_get_data(a) ((a)->ptr)
+#define BIO_set_init(a, init) ((a)->init = (init))
+#define BIO_set_shutdown(a, shut) ((a)->shutdown = (shut))
+#define BIO_get_shutdown(a) ((a)->shutdown)
+#endif
+
+struct Data {
```

Comment:
No, I agree. Can you help me come up with the right name?
How are the other symbols, data->fd and data->dst?